### PR TITLE
Feat/build results alex

### PR DESCRIPTION
**Pull request by khyberspache**

### CODE REVIEW

File: apps/build/src/components/results/results.module.css [hunk 1]
```diff
@@ -0,0 +1,221 @@
+.results {
```
Comment (kenrick) [inline apps/build/src/components/results/results.module.css:1]: nit: could you change the spacing in this file to 2 instead of 4 @khyberspache

**Review by kenrick — APPROVED** [review]
Small change then we can merge this

File: apps/build/src/components/results/results.module.css (hunk 1 above)
Comment (stephanwampouille) [reply apps/build/src/components/results/results.module.css:1]: I also think we could add overflow: auto to this, otherwise when you expand too many tests it adds overflow to the greater container - shifting the content

Comment (khyberspache) [reply apps/build/src/components/results/results.module.css:1]: I swear I thought it was 2 spaces

Comment (kenrick) [reply apps/build/src/components/results/results.module.css:1]: Good idea @stephanwampouille !
